pipeline/e2e-test-publish-results.yaml

```
@@ -9,8 +10,10 @@ steps:
       displayName: publish e2e test results
       timeoutInMinutes: 3
 
-    - publish: '$(System.DefaultWorkingDirectory)/test-results/e2e'
-      artifact: '$(Agent.JobName)-screenshots-and-extended-logs'
+    - task: 1ES.PublishPipelineArtifact@1
+      inputs:
+          targetPath: '$(System.DefaultWorkingDirectory)/test-results/e2e'
+          artifactName: '$(Agent.JobName)-screenshots-and-extended-logs'
       condition: failed() # because the detailed chrome logs are ~150MB and uploading them takes ~30s
```

Optional change to comment
```suggestion
      condition: failed() # Detailed logs are big and slow to upload, so skip this step if no failures occurred
```